cmd/operator-sdk/generate/csv.go

```diff
@@ -306,3 +319,11 @@ func validateVersion(version string) error {
 	}
 	return nil
 }
+
+func (c *csvCmd) interactivePrompt() {
+	c.csvSubcommand.DisplayName = getOperatorDisplayName()
+	c.csvSubcommand.Keywords = getOperatorKeywords()
+	c.csvSubcommand.Description = getOperatorDescription()
+	c.csvSubcommand.ProviderName = getProviderName()
+	c.csvSubcommand.ProviderURL = getProviderURL()
```

Comment:
If you create the generic `getString(msg)` and `getStringArray(msg)` then these calls would look more like:
```golang
    c.csvSubcommand.DisplayName = getString("Displayname for the operator: ")
    c.csvSubcommand.Keywords = getStringArray("List of keywords for your operator in the format :- word1, word2, word3 ... : ")
    c.csvSubcommand.Description = getString("Description for the operator: ")
    c.csvSubcommand.ProviderName = getString("The name of the publishing entity behind the operator: ")
    c.csvSubcommand.ProviderURL = getString("Any relevant URL for the provider name (if exists): ")
```